app/code/Magento/Security/Model/AdminSessionsManager.php

```diff
@@ -290,22 +304,25 @@ public function cleanExpiredSessions()
      */
     protected function createNewSession()
     {
-        $this->adminSessionInfoFactory
+        $adminSessionInfo = $this->adminSessionInfoFactory
             ->create()
             ->setData(
                 [
-                    'session_id' => $this->authSession->getSessionId(),
                     'user_id' => $this->authSession->getUser()->getId(),
                     'ip' => $this->remoteAddress->getRemoteAddress(),
                     'status' => AdminSessionInfo::LOGGED_IN
                 ]
             )->save();
 
+        $this->authSession->setAdminSessionInfoId($adminSessionInfo->getId());
+
         return $this;
     }
 
     /**
-     * @return \Magento\Security\Model\ResourceModel\AdminSessionInfo\Collection
+     * Retrieve new instance of admin session info collection
+     *
+     * @return Collection
```

Comment:
Can you please add a full class reference here as we are following the same in Core and it's easy to identify class.